compiler/rustc_middle/src/ty/ty_.rs

```diff
@@ -0,0 +1,9 @@
+use crate::ty::{Interned, TyKind, WithCachedTypeInfo};
```

Comment:
there shouldn't be both an sty and ty i think lol

Reply:
Yeah, I'll work on `sty` after this PR 😅